_security/multi-tenancy/tenant-index.md

```diff
@@ -16,7 +16,7 @@ redirect_from:
 - **Private** -- This tenant is exclusive to each user and can't be shared. It does not allow you to access routes or index patterns created by the user's global tenant.
 - **Custom** -- Administrators can create custom tenants and assign them to specific roles. Once created, these tenants can then provide spaces for specific groups of users.
 
-The global tenant is not a *primary* tenant in the sense that it replicates its content in a private tenant. To the contrary, if you make a change to your global tenant, you won't see that change reflected in your private tenant. Some example changes include the following:
+The global tenant in OpenSearch Dashboards doesn't synchronize its content with private tenants. When you make modifications within your global tenant, these changes are exclusive to the global tenant. They aren't automatically mirrored or replicated in the private tenant. Some example changes include the following:
```

Comment:
```suggestion
The global tenant in OpenSearch Dashboards doesn't synchronize its content with private tenants. When you make modifications within your global tenant, these changes are exclusive to the global tenant. They aren't automatically mirrored or replicated in the private tenant. Some example changes that are exclusive to the global tenant include the following:
```

Reply:
Hi @Naarcha-AWS, the operations listed:
Create visualizations
Create index patterns
Are not exclusive to only the global tenant so I think the new wording will be misleading.

Reply:
@leanneeliatra: I removed that part of the sentence. Let me know if this looks okay.

Reply:
I'm not sure how to 'mark the requested changes are addressed', but once that is complete, this all looks good to be merged now. Thanks!